extension-manifest-v3/src/store/options.js

```diff
@@ -167,19 +234,19 @@ async function migrateFromMV2() {
 }
 
 export async function observe(property, fn) {
-  let value;
-  const wrapper = async (options) => {
-    if (value === undefined || options[property] !== value) {
-      const prevValue = value;
-      value = options[property];
-
-      try {
+  let wrapper;
+  if (property) {
```

Comment:
we really need unit tests for this one - it is impossible to tell how it works or what we change. This is not a critic of the change, I simply incapable to judge its effect without use cases being listed

Reply:
I am thinking of making API like this inside of the `hybrids`, so it might not be needed here anymore. I would postpone it to a separate issue.